google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/CheckConsistencyParams.java

```diff
@@ -0,0 +1,36 @@
+package com.google.cloud.bigtable.admin.v2.models;
+
+import com.google.bigtable.admin.v2.TableName;
+
+public class CheckConsistencyParams {
```

Comment:
Please use AutoValue for small immutable value classes:
https://github.com/google/auto/blob/main/value/userguide/index.md